app.js

```diff
@@ -0,0 +1,24 @@
+const btnTheme = document.querySelector(".btn-theme");
+const htmlElement = document.documentElement;
+const moonIcon = document.querySelector(".btn-theme .fa-moon");
+
+btnTheme.addEventListener("click", () => {
+  if (!htmlElement.hasAttribute("data-theme")) {
+    htmlElement.setAttribute("data-theme", "dark");
+    localStorage.setItem("marvelsQuizTheme", "dark");
+  } else {
+    htmlElement.removeAttribute("data-theme");
+    localStorage.removeItem("marvelsQuizTheme");
+  }
+
+  moonIcon.classList.toggle("fas");
+});
+
+(() => {
```

Comment:
why is this inside an IIFE?

Reply:
As, I am storing the chosen theme, that the user chooses, in the local storage. So when the user comes back to the site, or moves to some other page, I am checking if it has a preferred dark theme, then do changes accordingly.

And as this has to be known as the page loads, and only one time, that's why I have used IIFE.